lib/ansible/plugins/lookup/grafana_dashboard.py

```diff
@@ -0,0 +1,177 @@
+# python 3 headers, required if submitting to Ansible
```

Comment:
Please replace this line with copyright and GPL license. See other lookup plugins for examples.